src/engine/Server.cpp

```diff
@@ -1003,10 +1078,19 @@ Awaitable<void> Server::processUpdate(
         return index_.deltaTriplesManager().modify<nlohmann::json>(
             [this, &params, &update, &requestTimer, &timeLimit, &messageSender,
              &cancellationHandle](auto& deltaTriples) {
+              LOG(INFO) << "Processing the following SPARQL update:\n"
+                        << update << std::endl;
```

Comment:
Isn't that `LOG` redundant because it is repeated inside the `createQueryExecutionContext` function?